src/lib/Support/SafeNames.cpp

```diff
@@ -200,6 +200,17 @@ class SafeNames::Impl
                     return getReserved(t);
                 }
 
+                if constexpr(T::isAlias())
+                {
+                    MRDOCS_ASSERT(! t.Name.empty());
+                    return t.Name;
+                }
+
+                if constexpr(T::isUsing()) {
+                    MRDOCS_ASSERT(! t.Name.empty());
```

Comment:
Safe names for using declarations are hard... I'll have to look into this

Reply:
👍 
I'll hold off on making further changes to that part until we have your input. Leaving this thread unresolved as a reminder.